src/services/mouse.js

```diff
@@ -57,6 +57,12 @@ Controller.open(function(_) {
       e.preventDefault(); // doesn't work in IE≤8, but it's a one-line fix:
       e.target.unselectable = true; // http://jsbin.com/yagekiji/1
 
+      // if textarea is not focusable so the controller should not look for
```

Comment:
@jack-vo I totally agree with @jeznag, all the rest looks good. Just remember that you should merge it in MathQuill first and then before merging in Q API and QE, you should update the current commit hash again to point to the merge commit from Mathquill repo.